hri_safety_sense/src/JoystickHandler.cpp

```diff
@@ -82,18 +82,20 @@ uint32_t JoystickHandler::handleNewMsg(const VscMsgType &incomingMsg)
     sendLeftMsg.header.stamp = this->nodeClock->get_clock()->now();
     sendLeftMsg.header.frame_id = this->frameId;
 
-    sendLeftMsg.axes.push_back((float)getStickValue(joyMsg->leftX));
-    sendLeftMsg.axes.push_back((float)getStickValue(joyMsg->leftY));
-    sendLeftMsg.axes.push_back((float)getStickValue(joyMsg->leftZ));
+    const float AXIS_MAX = 1023.;
```

Comment:
Please make this either `static` or (preferably) a `const` class variable to avoid re-allocating.

Reply:
Done in7e53f6d